app/config/config.yml

```diff
@@ -48,29 +48,25 @@ assetic:
             compress: true
             apply_to: "\.less$"
         cssrewrite: ~
-        sass:
-            bin: /usr/bin/sass
-            apply_to: "\.scss$"
-            style: compressed
+        scssphp:
+            formatter: 'Leafo\ScssPhp\Formatter\Compressed'
 
     assets:
         integrated_css:
             inputs:
                 - @IntegratedContentBundle/Resources/public/sass/main.scss
             filters:
-                - sass
+                - scssphp
             output: css/main.css
         integrated_js:
             inputs:
                 - @IntegratedFormTypeBundle/Resources/public/components/smalot-bootstrap-datetimepicker/js/bootstrap-datetimepicker.min.js
-                - @IntegratedFormTypeBundle/Resources/public/components/select2/select2.js
```

Comment:
@joenivl because it is already here: https://bitbucket.org/eactive/integrated-content-bundle/src/824eedc592736d67203bcc4a86eeb332021f548d/Resources/views/base.html.twig?at=0.5&fileviewer=file-view-default#base.html.twig-109